src/components/dropdown/dropdown_icon/icons/left_icon.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const LeftIcon = ({ className }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="20"
+    height="20"
+    viewBox="0 0 24 24"
```

Comment:
width="20"
height="20"
viewBox="0 0 24 24"

should the size be the same for here?

Reply:
It actually does not looks well when we set them in the viewBox as well